src/pybert/threads/optimization.py

```diff
@@ -136,15 +180,37 @@ def coopt(pybert) -> tuple[list[float], float, float, bool]:  # pylint: disable=
     else:
         peak_mags = array([0])
 
+    # Calculate Rx FFE tap weight candidates.
+    n_rx_weights = len(rx_taps)
+    n_enabled_rx_weights = len(list(filter(lambda t: t.enabled, rx_taps)))
+    if pybert.use_mmse:
+        rx_weightss = [[0.0] * n_rx_weights,]
+    else:
+        try:
+            rx_weightss = mk_tx_weights([[0.0] * n_rx_weights,], list(enumerate(rx_taps)))
+        except ValueError as err:
```

Comment:
Disabling "Use MMSE" and no other settings, we prompt the user this is 15106201 trials which they can approve but this will just bomb out on it.

```
[2025-07-14 12:02:45.450231]: PyBERT: Total number of combinations (439 M) is too great!
Sorry, that's more Rx FFE tap weight combinations than I can handle.
I had to abort the EQ optimization in your stead.
```

Reply:
Yeah, I struggled w/ this decision.
Are you of the opinion that if the user says, "yes, do it," then we should just do it?